src/graph/optimizer/rule/GetEdgesTransformRule.cpp

```diff
@@ -32,18 +32,20 @@ GetEdgesTransformRule::GetEdgesTransformRule() {
 }
 
 const Pattern &GetEdgesTransformRule::pattern() const {
-  static Pattern pattern =
-      Pattern::create({PlanNode::Kind::kAppendVertices, PlanNode::Kind::kLimit},
-                      {Pattern::create(PlanNode::Kind::kTraverse,
-                                       {Pattern::create(PlanNode::Kind::kScanVertices)})});
+  static Pattern pattern = Pattern::create(
```

Comment:
Note:
AV was removed here because I found that when handling `match ()-[e]->() return e, e limit 3` in this RULE transform, the limit count is lost in all operators, thus it will end up cannot be pushdown with SE(limit=-1).
When AV is removed from here, the other rule GetEdgesTransformAppendVerticesLimitRule will take care of the cases with AV.

Also, `match ()-[e]->() return e limit 3` is quite different from `match ()-[e]->() return e,e limit 3`